scripts/apps/archive/controllers/UploadController.ts

```diff
@@ -328,18 +335,32 @@ export function UploadController(
                 notify.error(message);
             }
 
-            if (invalidFiles.length > 0) {
-                notify.error(
-                    gettext(
-                        `The image you\'re trying to add is smaller than
-                        {{width}} x {{height}} pixels. Please use another one. File name: {{filename}}`,
+            if (invalidImages.length > 0) {
+                const images = invalidImages.map((image) => {
+                    return gettext(
+                        '<li>The size of {{name}} is {{width}}x{{height}}</li>',
                         {
-                            width: appConfig.pictures.minWidth,
-                            height: appConfig.pictures.minHeight,
-                            filename: invalidFiles.join(', '),
+                            name: image.name,
+                            width: image.width,
+                            height: image.height,
+                        },
+                    );
+                });
+
+                modal.alert({
```

Comment:
Can you use [showModal](https://github.com/superdesk/superdesk-client-core/blob/develop/scripts/core/services/modalService.tsx#L7) when displaying HTML content?